corehq/apps/hqadmin/service_checks.py

```diff
@@ -61,23 +62,42 @@ def check_redis():
         return ServiceStatus(False, "Redis is not configured on this system!")
 
 
-def check_rabbitmq():
-    if settings.BROKER_URL.startswith('amqp'):
-        amqp_parts = settings.BROKER_URL.replace('amqp://', '').split('/')
+def check_all_rabbitmq():
+    unwell_rabbits = []
+    ip_regex = re.compile(r'[0-9]+.[0-9]+.[0-9]+.[0-9]+')
+
+    for broker_url in settings.BROKER_URL.split(';'):
```

Comment:
I think we should have used the list of URLs instead of supporting the `;` syntax https://docs.celeryproject.org/en/latest/userguide/configuration.html#broker-url

Would have made the configuration and this code easier to read. Alternatively, I wonder if there's a `get_brokers` utility that celery provides to make this simple